.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+	"bracketSpacing": false,
+	"singleQuote": true
+}
```

Comment:
This should not be here. I don't use Prettier and shouldn't have to have boilerplate config to disable it either.